CVSROOT:	/cvs/mesa
Module name:	Mesa
Repository:	Mesa/src/mesa/x86-64/
Changes by:	idr@gabe.	05/12/01 16:25:00

Log message:
  Fix build issues on x86-64 due to missing include path in
  glapi_x86-64.S.
  
  Bug: #5602

Modified files:
      Mesa/src/mesa/glapi/:	Tag: mesa_6_4_branch
        gl_x86-64_asm.py 
      Mesa/src/mesa/x86-64/:	Tag: mesa_6_4_branch
        glapi_x86-64.S 
  
  Revision      Changes    Path
  1.4.2.2       +1 -1      Mesa/src/mesa/glapi/gl_x86-64_asm.py
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/glapi/gl_x86-64_asm.py
  1.4.2.2       +1 -1      Mesa/src/mesa/x86-64/glapi_x86-64.S
  http://cvs.freedesktop.org/mesa/Mesa/src/mesa/x86-64/glapi_x86-64.S

Index: src/mesa/x86-64/glapi_x86-64.S
===================================================================
RCS file: /cvs/mesa/Mesa/src/mesa/x86-64/glapi_x86-64.S,v
retrieving revision 1.5
retrieving revision 1.6
diff -u -b -B -r1.5 -r1.6
--- src/mesa/x86-64/glapi_x86-64.S	2 Nov 2005 20:59:37 -0000	1.5
+++ src/mesa/x86-64/glapi_x86-64.S	2 Dec 2005 00:25:06 -0000	1.6
@@ -29,7 +29,7 @@
  * the symbol visibility mode to 'default'.
  */
 
-#include "assyntax.h"
+#include "../x86/assyntax.h"
 
 #if defined(__GNUC__) && (__GNUC__ * 100 + __GNUC_MINOR__) >= 303
 #  pragma GCC visibility push(default)
